docs_src/getting-started/quick-start/index.md

```diff
@@ -232,4 +236,4 @@ You will begin seeing your random number readings appear in the Messages area on
 
 Congratulations! You now have a full EdgeX deployment reading data from a (virtual) device and publishing it to an MQTT broker in the cloud, and you were able to control your device through commands into EdgeX. 
 
-It's time to continue your journey by reading the [Introduction](../../index.md) to EdgeX Foundry, what it is and how it's built. From there you can take the [Walkthrough](../../walk-through/Ch-Walkthrough.md) to learn how the microservices work together to control devices and read data from them as you just did.
+It's time to continue your journey by reading the [Introduction](../../index.md) to EdgeX Foundry, what it is and how it's built. From there you can take the [Walkthrough](../../walk-through/Ch-Walkthrough.md) to learn how the micro services work together to control devices and read data from them as you just did.
```

Comment:
Microservices is one word.
https://en.wikipedia.org/wiki/Microservices

Reply:
No - we attempted to be consistent throughout the documentation to keep it two words.  See the reference.https://docs.edgexfoundry.org/2.0/general/Definitions/#micro-service.  I spotted this inconsistency on this page and changed it.

Reply:
But it is a single word, not two, everywhere you look on the internet. It is wrong in the Definitions.

Reply:
IMHO, we need to fix all these throughout the docs to be correctly one word. This obviously is not the PR for that... ;-) So I don't care right now how it is in this PR. :-)